source/features/useful-not-found-page.tsx

```diff
@@ -118,4 +132,11 @@ void features.add({
 	],
 	repeatOnAjax: false,
 	init
+}, {
+	include: [
+		pageDetect.isPRCommit
+	],
```

Comment:
```suggestion
	],
	exclude: [
		() => !document.title.startsWith('Commit range not found · Pull Request')
	],
```

Then there is no need for elementready

Reply:
@yakov116 It's a good idea to detect the page via `document.title`. I changed the `include` instead, though. Do you think we do have to combine that with `isPRCommit` to prevent false positives?

I didn't drop `elementReady`, because it prevents the page from changing after it has loaded. I don't know if that's actually necessary here, but it also doesn't hurt.

Reply:
Its good to have it as an includes and then exclude if it does not match. Since there is no way to know that `document.title.startsWith('Commit range not found etc` = isPRCoomit

Reply:
It is, since it says *PR* and *commit* :wink: 

But there could theoretically be an issue, PR or commit that causes the title to start with that exact string.

Reply:
Could be. But that how we did all excludes until now.

Reply:
I guess you're right, it doesn't hurt to be explicit :)